### PR TITLE
Projection bounds

### PR DESCRIPTION
**Pull request by bbuzz31**
This adds support for reusing a weather model file that is in projected coordinates. 
I.e., if HRRR is already downloaded, and you pass a bounding box in lat/lon, it will mismatch because HRRR is in meters.

### CODE REVIEW

File: tools/RAiDER/models/weatherModel.py [hunk 1]
```diff
@@ -439,10 +439,16 @@ def checkContainment(self: weatherModel,
            and False otherwise.
         """
         ymin_input, ymax_input, xmin_input, xmax_input = ll_bounds
-        input_box = box(xmin_input, ymin_input, xmax_input, ymax_input)
+        input_box   = box(xmin_input, ymin_input, xmax_input, ymax_input)
         xmin, ymin, xmax, ymax = self.bbox
         weather_model_box = box(xmin, ymin, xmax, ymax)
 
+        world_box  = box(-180, -90, 180, 90)
```
Comment (jlmaurer) [inline tools/RAiDER/models/weatherModel.py:446]: I don't think you need the if-statement, just transform everytime. We should just make sure to add a small buffer to the download region

File: tools/RAiDER/models/hrrr.py [hunk 2]
```diff
@@ -269,6 +269,8 @@ def _download_hrrr_file(self, DATE, out, model='hrrr', product='prs', fxx=0, ver
             save_dir=Path(os.path.dirname(out)),
         )
         pf = H.download(":(SPFH|PRES|TMP|HGT):", verbose=verbose)
+        # breakpoint()
```
Comment (jlmaurer) [inline tools/RAiDER/models/hrrr.py:272]: is this needed?

File: tools/RAiDER/models/weatherModel.py (hunk 1 above)
Comment (bbuzz31) [reply tools/RAiDER/models/weatherModel.py:446]: Do all the weather models have a projection though?

File: tools/RAiDER/models/hrrr.py (hunk 2 above)
Comment (bbuzz31) [reply tools/RAiDER/models/hrrr.py:272]: nope forgot to remove

File: tools/RAiDER/models/weatherModel.py (hunk 1 above)
Comment (dbekaert) [reply tools/RAiDER/models/weatherModel.py:446]: All models should come with projections.

Comment (bbuzz31) [reply tools/RAiDER/models/weatherModel.py:446]: I think we should still have the check that the user input is in geocoordinates before converting

Comment (jlmaurer) [reply tools/RAiDER/models/weatherModel.py:446]: Makes sense.

**Review by jlmaurer — APPROVED** [review]

**Comment by jlmaurer** [comment]
I'm going ahead and merging this so I can build on it.

**Pull request merged**